core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -304,4 +304,9 @@ public void onLoadTimeout() {
 
         void onBeat(NoPlayer player);
     }
+
+    interface DroppedVideoFramesListener {
+
+        void onDroppedVideoFrames(int droppedFrames);
```

Comment:
Will clients be measuring time between events themselves?

Reply:
I didn't find the metric particularly useful but I can add it

Reply:
It's fine, it was more of a question. Might be better for the client to handle it 😄